src/components/button/button.tsx

```diff
@@ -19,7 +19,8 @@ export interface Props extends BreakpointsProps {
     | 'text'
     | 'text-blue'
     | 'menu-link-secondary'
-    | 'link-blue';
+    | 'link-blue'
+    | 'link-gray';
```

Comment:
Hmm wouldn't it make more sense to call this the `text-gray` variant? What's the connection with links here?

Reply:
It's just a button type that is used in Figma. They have also "Text gray" button type but that's not what used here.

![image](https://github.com/user-attachments/assets/7f2ccc89-9b37-4e14-890b-f2f6b1fdc93d)